y_lb.libraries.yml

```diff
@@ -65,7 +70,7 @@ header:
     assets/js/header.js: { minified: false }
 
 main_menu:
-  version: 1.13
+  version: 1.14
```

Comment:
See https://yusaslackinstance.slack.com/archives/C03E6SQQ0Q5/p1726007392209179

Reply:
@rollki I believe change is correct and follows the right convention. The only issue I see here is that the `layout_builder` library above should also be incremented.

Reply:
Yes, the change is correct, I wanted David to be aware, for the future :)